solutions/cf-919D.mdx

```diff
@@ -47,7 +47,7 @@ using pi = pair<int,int>;
 void setIO(string name = "") {
 	cin.tie(0)->sync_with_stdio(0); // see /general/fast-io
 	if (sz(name)) {
-		(void)!freopen((name+".in").c_str(), "r", stdin); // see /general/io
+		(void)!freopen((name+".in").c_str(), "r", stdin); // see /general/input-output
```

Comment:
maybe refactor all of these to use the codesnip instead? so they don't all need to be changed when the codesnip is updated